lib/rubocop/cop/style/space_around_operators.rb

```diff
@@ -64,6 +64,16 @@ def check_operator(op)
             add_offense(with_space, op,
                         'Surrounding space missing for operator' \
                         " '#{op.source}'.")
+          elsif with_space.source =~ /(^  |  $)/ && !multi_space_operator?(op)
+            add_offense(with_space, op,
+                        'Too many spaces around operator' \
```

Comment:
I'd prefer "Operator X should be surrounded with a single space". Sounds clearer to me.